ui/src/components/SampleView/MotorControls.jsx

```diff
@@ -25,21 +25,22 @@ function MotorControls() {
   );
 
   if (!verticalMotorProps || !horizontalMotorProps) {
-    return motorsProps.map(({ attribute: k, role }) => (
-      <MotorInput key={k} role={role} />
-    ));
+    return motorsProps.map(({ role }) => <MotorInput key={role} role={role} />);
   }
 
   return (
     <>
-      {motorsProps.slice(0, 3).map(({ attribute: k, role }) => (
-        <MotorInput key={k} role={role} />
+      {motorsProps.slice(0, 3).map(({ role }) => (
+        <MotorInput key={role} role={role} />
       ))}
 
-      <TwoAxisTranslationControl
-        verticalMotorProps={verticalMotorProps}
-        horizontalMotorProps={horizontalMotorProps}
-      />
+      <div className={motorInputStyles.container}>
+        <p className={`${motorInputStyles.label} mb-1`}>Sample alignment</p>
```

Comment:
I've moved the container and label out of `TwoAxisTranslationControl` for consistency with `OneAxisTranslationControl`.